content/src/helpers/FetcherFactory.ts

```diff
@@ -1,18 +1,24 @@
-import { Fetcher } from 'dcl-catalyst-commons'
+import { createFetchComponent } from '@well-known-components/fetch-component'
+import { IFetchComponent, RequestOptions } from '@well-known-components/interfaces'
+import ms from 'ms'
+import * as fetch from 'node-fetch'
```

Comment:
should we still import this?

Reply:
We do need it to decorate the fetcher, mostly because the Response (I don't want to cast that Response to another type). Anyways, these RequestInit and Response types could be exported from `interfaces` or even from `fetch-component`. Both ways, we will have node-fetch as a dep or chained dep.

The scope of this PR is just to stop using custom fetchers (and bumps new catalyst-client), I'm still thinking that there is room for improvements, for example: I think we don't need a FetcherFactory and we should support default options when creating the component fetcher. But this change will require a new version of fetch-component that I can release afterwards while integrating the latest version of `catalyst-client` (which currently is outdated by 3 major updates, that's why my eager to merge this PR :).

Please let me know if you think we should take care of these improvements on this PR.

Reply:
this is fine, I just though that interfaces was exposing this as well, not sure why is not doing so.

Reply:
I'll add these types there, thank you!